java-file-formats/word-library/find-item-in-word-document.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Find item in Word document in Java | Syncfusion
+description: Find an item in the Word document in Java using Syncfusion Java Word library (Essential DocIO)
+platform: java-file-formats
+control: Word Library
+documentation: UG
+---
+
+# Find item in Word document
+
+Just like you can search for a text in a Word document, you can also search for an item (like image, content control, textbox, and so on). The .NET Word (DocIO) library supports to find an item in Word document based on its properties. With this functionality, you can:
```

Comment:
The Java Word library supports to find a..

Reply:
Adressed